### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by jlpedrosa**
Everything that was on the other repo for the helm chart + some of the extra files that you had on the other repo,

### CODE REVIEW

File: .github/workflows/helm_release.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
```
Comment (Koenkk) [inline .github/workflows/helm_release.yaml:6]: This should be `main` now

Comment (jlpedrosa) [inline .github/workflows/helm_release.yaml:6]:
```suggestion
      - main
```

**Comment by pmarques** [comment]
Thank you @jlpedrosa for creating the charts, just in time as I move my z2m instance into k8s.

I have some suggestions after this merge, like adding support for secrets. I tweaked the chart to mount the secret(s) as a file and use it as described in [Advanced Configuration](https://www.zigbee2mqtt.io/guide/configuration/frontend.html#advanced-configuration) which us support in a few other places.

```
mqtt:
  password: '!secrets.yaml mqtt_password'
```

**Comment by jlpedrosa** [comment]
> Thank you @jlpedrosa for creating the charts, just in time as I move my z2m instance into k8s.
> 
> I have some suggestions after this merge, like adding support for secrets. I tweaked the chart to mount the secret(s) as a file and use it as described in [Advanced Configuration](https://www.zigbee2mqtt.io/guide/configuration/frontend.html#advanced-configuration) which us support in a few other places.
> 
> ```
> mqtt:
>   password: '!secrets.yaml mqtt_password'
> ```

We need to be careful with those ones, there are files that zigbee2mqtt will try to modify, so if you create a config map or a secret (are readonly) it will start failing.

**Comment by pmarques** [comment]
> there are files that zigbee2mqtt will try to modify, so if you create a config map or a secret (are readonly) it will start failing

Indeed, I only run a couple of tests from scratch since I’ve migrated the configurations from another host. I like to keep some configs in the config map/secrets like the MQTT connection while others I don’t really have much of a preference, like pan_id etc.
At this point I’m not to bothered about the Front end trying to update any of the values but IIRC, for a new installation at lest the `GENERATE` values will fail to be written 🤔

**Comment by Koenkk** [comment]
@jlpedrosa thanks, I've configured https://charts.zigbee2mqtt.io/ to point to this repo. Can you check if everything is OK? Then we can merge the docs PR.

**Comment by jlpedrosa** [comment]
index.yaml is correct, the file can be downloaded, the install on the cluster is working:
```
kubectl get -n flux-system helmreleases.helm.toolkit.fluxcd.io zigbee2mqtt
NAME          AGE   READY   STATUS
zigbee2mqtt   77s   True    Helm install succeeded for release homeassistant/homeassistant-zigbee2mqtt.v1 with chart zigbee2mqtt@1.37.1

 kubectl get -n homeassistant all                                          
NAME                            READY   STATUS    RESTARTS   AGE
pod/esphomerel-0                1/1     Running   0          5d22h
pod/homeassistant-0             1/1     Running   0          25m
pod/mosquitto-9856877db-xkw7q   1/1     Running   0          5d22h
pod/zigbee2mqtt-0               1/1     Running   0          80s

NAME                    TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
service/esphomerel      LoadBalancer   10.80.59.70    10.70.0.4     80:31058/TCP     5d22h
service/homeassistant   LoadBalancer   10.80.135.13   10.70.0.8     8080:30972/TCP   5d22h
service/mosquitto       LoadBalancer   10.80.171.40   10.70.0.9     1883:30576/TCP   5d22h
service/zigbee2mqtt     LoadBalancer   10.80.139.1    10.70.0.18    8080:31686/TCP   80s

NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/mosquitto   1/1     1            1           5d22h

NAME                                  DESIRED   CURRENT   READY   A
![image](https://github.com/Koenkk/zigbee2mqtt-chart/assets/4932947/8023b84d-4ac6-4252-a050-72fddafddb5a)
GE
replicaset.apps/mosquitto-9856877db   1         1         1       5d22h

NAME                             READY   AGE
statefulset.apps/esphomerel      1/1     5d22h
statefulset.apps/homeassistant   1/1     5d22h
statefulset.apps/zigbee2mqtt     1/1     80s
```

System is up and running, also through ingress:
image

**Pull request merged**